projects/schematics/src/add-spartacus/spartacus-features.ts

```diff
@@ -41,30 +41,26 @@ function configureSpartacusModules(
         .getFilePath()
         .includes(`${SPARTACUS_FEATURES_MODULE}.module.ts`)
     ) {
-      [
-        `// Auth Core
```

Comment:
We are removing these comments, as it's not easy to handle them in the customers app - when ordering feature modules, they are removed)